tree/dataframe/src/RDataFrame.cxx

```diff
@@ -814,6 +814,65 @@ Without this, two partial histograms resulting from two distributed tasks would
 to errors when merging them. Failing to pass a histogram model will raise an error on the client side, before starting
 the distributed execution.
 
+### Live Visualization in Distributed Mode with Dask
+
+The Live Visualization feature allows real-time data representation of plots generated during the execution 
+of a distributed RDataFrame application. 
+It enables visualizing intermediate results as they are computed across multiple nodes of a Dask cluster
+by creating a canvas and continuously updating it as partial results become available. 
+
+The LiveVisualize() function should be imported from the DistRDF module:
+
+~~~{.py}
+from DistRDF import LiveVisualize
```

Comment:
We should show the same importing pattern as previous paragraphs of this distributed section. The `DistRDF`  module per se is an implementation detail and users shouldn't need to know about it directly. What we do instead is showcase it as a submodule of ROOT. In this case, we can write
```suggestion
import ROOT
LiveVisualize = ROOT.RDF.Experimental.Distributed.LiveVisualize
```